ScheduledExecutorServiceBasicTest.schedule_withMapChanges_durable

https://hazelcast-l337.ci.cloudbees.com/job/new-lab-fast-pr/13829
```
java.lang.AssertionError: expected:<98644> but was:<98643>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at com.hazelcast.scheduledexecutor.ScheduledExecutorServiceBasicTest.schedule_withMapChanges_durable(ScheduledExecutorServiceBasicTest.java:342)
```

http://jenkins.hazelcast.com/view/Official%20Builds/job/Hazelcast-3.x-CorrettoJDK8/47/testReport/com.hazelcast.scheduledexecutor.impl/ScheduledExecutorServiceBasicTest/schedule_withMapChanges_durable/

**Stacktrace**
```
com.hazelcast.spi.exception.ResponseAlreadySentException: NormalResponse already responseReceived for callback: Invocation{op=com.hazelcast.cp.internal.operation.unsafe.UnsafeRaftReplicateOp{serviceName='hz:core:raft', identityHash=1064624952, partitionId=112, replicaIndex=0, callId=-28, invocationTime=1577260849578 (2019-12-25 08:00:49.578), waitTimeout=-1, callTimeout=60000, groupId=CPGroupId{name='waitAfterStartLatch', seed=0, groupId=112}, op=com.hazelcast.cp.internal.datastructures.countdownlatch.operation.AwaitOp{serviceName='hz:raft:countDownLatchService', name=waitAfterStartLatch, invocationUid=120283be-8262-4077-a8d8-b323344e38ed, timeoutMillis=60000, callerAddress=[127.0.0.1]:5702, callId=28}}, tryCount=250, tryPauseMillis=500, invokeCount=15, callTimeoutMillis=60000, firstInvocationTimeMs=1577260848956, firstInvocationTime='2019-12-25 08:00:48.956', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5702, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=MockConnection{localEndpoint=[127.0.0.1]:5702, remoteEndpoint=[127.0.0.1]:5701, alive=false}}, current-response: true
	at java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:627)
	at com.hazelcast.spi.impl.AbstractInvocationFuture.tryWrapInSameClass(AbstractInvocationFuture.java:1964)
	at com.hazelcast.spi.impl.AbstractInvocationFuture.wrapRuntimeException(AbstractInvocationFuture.java:1940)
	at com.hazelcast.spi.impl.AbstractInvocationFuture.wrapOrPeel(AbstractInvocationFuture.java:1921)
	at com.hazelcast.spi.impl.AbstractInvocationFuture$ExceptionalResult.wrapForJoinInternal(AbstractInvocationFuture.java:1395)
	at com.hazelcast.spi.impl.AbstractInvocationFuture.resolveAndThrowForJoinInternal(AbstractInvocationFuture.java:592)
	at com.hazelcast.spi.impl.AbstractInvocationFuture.joinInternal(AbstractInvocationFuture.java:576)
	at com.hazelcast.cp.internal.datastructures.countdownlatch.proxy.CountDownLatchProxy.countDown(CountDownLatchProxy.java:69)
	at com.hazelcast.scheduledexecutor.impl.ScheduledExecutorServiceBasicTest.schedule_withMapChanges_durable(ScheduledExecutorServiceBasicTest.java:787)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:114)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:106)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.lang.Thread.run(Thread.java:748)
Caused by: com.hazelcast.spi.exception.ResponseAlreadySentException: NormalResponse already responseReceived for callback: Invocation{op=com.hazelcast.cp.internal.operation.unsafe.UnsafeRaftReplicateOp{serviceName='hz:core:raft', identityHash=1064624952, partitionId=112, replicaIndex=0, callId=-28, invocationTime=1577260849578 (2019-12-25 08:00:49.578), waitTimeout=-1, callTimeout=60000, groupId=CPGroupId{name='waitAfterStartLatch', seed=0, groupId=112}, op=com.hazelcast.cp.internal.datastructures.countdownlatch.operation.AwaitOp{serviceName='hz:raft:countDownLatchService', name=waitAfterStartLatch, invocationUid=120283be-8262-4077-a8d8-b323344e38ed, timeoutMillis=60000, callerAddress=[127.0.0.1]:5702, callId=28}}, tryCount=250, tryPauseMillis=500, invokeCount=15, callTimeoutMillis=60000, firstInvocationTimeMs=1577260848956, firstInvocationTime='2019-12-25 08:00:48.956', lastHeartbeatMillis=0, lastHeartbeatTime='1970-01-01 00:00:00.000', target=[127.0.0.1]:5702, pendingResponse={VOID}, backupsAcksExpected=-1, backupsAcksReceived=0, connection=MockConnection{localEndpoint=[127.0.0.1]:5702, remoteEndpoint=[127.0.0.1]:5701, alive=false}}, current-response: true
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.sendResponse(Invocation.java:218)
	at com.hazelcast.spi.impl.operationservice.Operation.sendResponse(Operation.java:470)
	at com.hazelcast.cp.internal.RaftService.completeFutures(RaftService.java:1229)
	at com.hazelcast.cp.internal.datastructures.spi.blocking.AbstractBlockingService.completeFutures(AbstractBlockingService.java:321)
	at com.hazelcast.cp.internal.datastructures.spi.blocking.AbstractBlockingService.notifyWaitKeys(AbstractBlockingService.java:316)
	at com.hazelcast.cp.internal.datastructures.countdownlatch.CountDownLatchService.countDown(CountDownLatchService.java:57)
	at com.hazelcast.cp.internal.datastructures.countdownlatch.operation.CountDownOp.run(CountDownOp.java:53)
	at com.hazelcast.cp.internal.operation.unsafe.AbstractUnsafeRaftOp.call(AbstractUnsafeRaftOp.java:65)
	at com.hazelcast.spi.impl.operationservice.impl.OperationRunnerImpl.call(OperationRunnerImpl.java:220)
	at com.hazelcast.spi.impl.operationservice.impl.OperationRunnerImpl.run(OperationRunnerImpl.java:209)
	at com.hazelcast.spi.impl.operationexecutor.impl.OperationThread.process(OperationThread.java:151)
	at com.hazelcast.spi.impl.operationexecutor.impl.OperationThread.process(OperationThread.java:129)
	at com.hazelcast.spi.impl.operationexecutor.impl.OperationThread.run(OperationThread.java:114)
```

--- a/hazelcast/src/main/java/com/hazelcast/cp/internal/RaftService.java
+++ b/hazelcast/src/main/java/com/hazelcast/cp/internal/RaftService.java
@@ -72,6 +72,7 @@ import com.hazelcast.internal.partition.MigrationAwareService;
 import com.hazelcast.internal.partition.MigrationEndpoint;
 import com.hazelcast.internal.partition.PartitionMigrationEvent;
 import com.hazelcast.internal.partition.PartitionReplicationEvent;
+import com.hazelcast.internal.serialization.Data;
 import com.hazelcast.internal.services.GracefulShutdownAwareService;
 import com.hazelcast.internal.services.ManagedService;
 import com.hazelcast.internal.services.MembershipAwareService;
@@ -81,8 +82,8 @@ import com.hazelcast.internal.util.Clock;
 import com.hazelcast.internal.util.ExceptionUtil;
 import com.hazelcast.internal.util.executor.ManagedExecutorService;
 import com.hazelcast.logging.ILogger;
-import com.hazelcast.internal.serialization.Data;
 import com.hazelcast.spi.exception.PartitionMigratingException;
+import com.hazelcast.spi.exception.ResponseAlreadySentException;
 import com.hazelcast.spi.impl.InternalCompletableFuture;
 import com.hazelcast.spi.impl.NodeEngine;
 import com.hazelcast.spi.impl.NodeEngineImpl;
@@ -1225,9 +1226,7 @@ public class RaftService implements ManagedService, SnapshotAwareService<Metadat
             UnsafeModePartitionState unsafeModeState = unsafeModeStates[partitionId];
             for (Long index : indices) {
                 Operation op = unsafeModeState.removeWaitingOp(index);
-                if (op != null) {
-                    op.sendResponse(result);
-                }
+                sendOperationResponse(op, result);
             }
         }
         return true;
@@ -1255,12 +1254,20 @@ public class RaftService implements ManagedService, SnapshotAwareService<Metadat
             UnsafeModePartitionState unsafeModeState = unsafeModeStates[partitionId];
             for (Entry<Long, Object> result : results) {
                 Operation op = unsafeModeState.removeWaitingOp(result.getKey());
-                if (op != null) {
-                    op.sendResponse(result.getValue());
-                }
+                sendOperationResponse(op, result.getValue());
             }
         }
         return true;
+    }
+
+    private void sendOperationResponse(Operation op, Object result) {
+        if (op != null) {
+            try {
+                op.sendResponse(result);
+            } catch (ResponseAlreadySentException e) {
+                op.logError(e);
+            }
+        }
     }
 
     @Override

